README.md

```diff
@@ -10,7 +10,7 @@
 ### Installation
 
 * Requires Python `>=3.7`
-* `pip install firebolt-sdk`
```

Comment:
Should we do `"firebolt-sdk==0.*"` instead? It's just I feel uneasy relying on an alpha always being there.

Reply:
It makes sense

Reply:
Done